LinearRegression/LinearRegressionBackend/DataProvider/ImageProcess.cs

```diff
@@ -0,0 +1,77 @@
+﻿using DocumentFormat.OpenXml.Drawing.Charts;
+using MathNet.Numerics.LinearAlgebra;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Interop;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+
+namespace LinearRegressionBackend.DataProvider
+{
+    public class ImageProcess : IImageConverter
+    {
+        public  Vector<double> CreateLabel(string filename)
+        {
+            double[] labelArray = new double[3]; //in the case we have 3 shapes 
+
+            switch(filename.Remove(filename.Length - 1))
```

Comment:
We should use `if` and `elseif` statements. In the condition, check if the file name contains the desired label e.g. `filename.ToLower().Contains("circle")`. (Lowercase the filename for incase sensitivity)